packages/vm/src/runBlock.ts

```diff
@@ -331,11 +323,9 @@ async function assignWithdrawals(this: VM, block: Block): Promise<void> {
   const state = this.eei
   const withdrawals = block.withdrawals!
   for (const withdrawal of withdrawals) {
-    const { address: addressData, amount: amountData } = withdrawal
-    const address = new Address(toBuffer(addressData))
-    const amount = Buffer.isBuffer(amountData)
-      ? BigInt('0x' + amountData.toString('hex'))
-      : BigInt(amountData)
+    const { address, amount } = withdrawal
+    // skip touching account if no amount update
+    if (amount === 0n) continue
```

Comment:
Please no `n` notation for BigInt, at least in all libraries lower than client (client I guess it's not so critical).

Please switch to `BigInt()` (see argumentation I made on some PR before) and give this (n-notation usage) a general review on this PR.